### ISSUE TITLE
Clausal subject of положено?

### ISSUE THREAD

**Issue by ftyers** [UniversalDependencies/UD_Russian-SynTagRus#6]
While looking at the Russian constructicon, I was looking for examples of NP-Dat не положено VP-Inf, and I found this example. Should (положено, финансировать) be `csubj` not `nsubj` here ? Especially as it has an object, _культуру_. 

```
# source = 2014Kulturnye_olimpiitsy.xml 46
# text = По традиции культуру все равно положено финансировать, а в неосмысленном пространстве, без четкой системы координат, потоками управлять удобнее.
# sent_id = 36838
1       По      по      ADP     _       _       2       case    2:case  _
2       традиции        традиция        NOUN    _       Animacy=Inan|Case=Dat|Gender=Fem|Number=Sing    6       nmod    6:nmod  _
3       культуру        культура        NOUN    _       Animacy=Inan|Case=Acc|Gender=Fem|Number=Sing    7       obj     7:obj   _
4       все     все     PRON    _       Animacy=Inan|Case=Nom|Gender=Neut|Number=Sing   6       advmod  6:advmod        _
5       равно   равный  ADJ     _       Degree=Pos|Gender=Neut|Number=Sing|Variant=Short        4       fixed   4:fixed _
6       положено        положенный      ADJ     _       Degree=Pos|Gender=Neut|Number=Sing|Variant=Short        0       root    0:root  _
7       финансировать   финансировать   VERB    _       Aspect=Imp|VerbForm=Inf|Voice=Act       6       nsubj   6:nsubj SpaceAfter=No
8       ,       ,       PUNCT   _       _       7       punct   7:punct _
9       а       а       CCONJ   _       _       21      cc      21:cc   _
10      в       в       ADP     _       _       12      case    12:case _
11      неосмысленном   неосмысленный   ADJ     _       Case=Loc|Degree=Pos|Gender=Neut|Number=Sing     12      amod    12:amod _
12      пространстве    пространство    NOUN    _       Animacy=Inan|Case=Loc|Gender=Neut|Number=Sing   21      nmod    21:nmod SpaceAfter=No
13      ,       ,       PUNCT   _       _       12      punct   12:punct        _
14      без     без     ADP     _       _       16      case    16:case _
15      четкой  четкий  ADJ     _       Case=Gen|Degree=Pos|Gender=Fem|Number=Sing      16      amod    16:amod _
16      системы система NOUN    _       Animacy=Inan|Case=Gen|Gender=Fem|Number=Sing    12      conj    12:conj _
17      координат       координата      NOUN    _       Animacy=Inan|Case=Gen|Gender=Fem|Number=Plur    16      nmod    16:nmod SpaceAfter=No
18      ,       ,       PUNCT   _       _       17      punct   17:punct        _
19      потоками        поток   NOUN    _       Animacy=Inan|Case=Ins|Gender=Masc|Number=Plur   20      obl     20:obl  _
20      управлять       управлять       VERB    _       Aspect=Imp|VerbForm=Inf|Voice=Act       21      nsubj   21:nsubj        _
21      удобнее удобный ADJ     _       Degree=Cmp      6       conj    6:conj  SpaceAfter=No
22      .       .       PUNCT   _       _       21      punct   21:punct        _

```

**Comment by ftyers** [comment]
Another example with стоит:

```
# source = 2003Nepotoplyaemye.xml 82
# text = "Не стоит ожидать в ближайшем будущем глобальных прорывов в этой области.
# sent_id = 2305
1       "       "       PUNCT   _       _       2       punct   2:punct SpaceAfter=No
2       Не      не      PART    _       _       3       advmod  3:advmod        _
3       стоит   стоить  VERB    _       Aspect=Imp|Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin|Voice=Act      0       root    0:root  _
4       ожидать ожидать VERB    _       Aspect=Imp|VerbForm=Inf|Voice=Act       3       nsubj   3:nsubj _
5       в       в       ADP     _       _       7       case    7:case  _
6       ближайшем       ближайший       ADJ     _       Case=Loc|Degree=Pos|Gender=Neut|Number=Sing     7       amod    7:amod  _
7       будущем будущее NOUN    _       Animacy=Inan|Case=Loc|Gender=Neut|Number=Sing   4       obl     4:obl   _
8       глобальных      глобальный      ADJ     _       Case=Gen|Degree=Pos|Number=Plur 9       amod    9:amod  _
9       прорывов        прорыв  NOUN    _       Animacy=Inan|Case=Gen|Gender=Masc|Number=Plur   4       obl     4:obl   _
10      в       в       ADP     _       _       12      case    12:case _
11      этой    этот    DET     _       Case=Loc|Gender=Fem|Number=Sing 12      amod    12:amod _
12      области область NOUN    _       Animacy=Inan|Case=Loc|Gender=Fem|Number=Sing    9       nmod    9:nmod  SpaceAfter=No
13      .       .       PUNCT   _       _       12      punct   12:punct        _

```

**Comment by dan-zeman** [comment]
I vote for infinitives to be considered clauses in Slavic languages. (That is also what we have in Czech.) Then yes, `csubj` should be used here instead of `nsubj`.

**Comment by Kira-D** [comment]
fixed in 2.1
